### PR TITLE
Boost/update/image guide beta

### PR DESCRIPTION
**Pull request by pyronaur**
Quality of life improvements for the image guide feature

#### Changes proposed in this Pull Request:
Fixed typos, formatted files, and added a checkmark for some image sizes to indicate that they're okay. Added a documentation link.

#### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:

1. Apply patch
2. Build Jetpack Boost
3. Check out the changes mentioned above

### CODE REVIEW

File: projects/plugins/boost/app/features/image-guide/src/ui/Popup.svelte [hunk 1]
```diff
@@ -51,23 +53,29 @@
...
-				</div>
-			{:else if ratio > 1 && ratio < 1.3}
-				<div class="explanation">
-					Images may not always be perfectly sized on all screen sizes, so this is probably ok The
-					image loaded is <strong>{ratio}x</strong> larger than it appears in the browser.
+					The image size is very close to the size it appears in the browser.
+					{#if ratio > 1}
+						Because there are various screen sizes, it's okay for the image to be <strong
+							>{ratio}x</strong
+						> than it appears on the page.
```
Comment (dilirity) [inline projects/plugins/boost/app/features/image-guide/src/ui/Popup.svelte]: Not sure what's going on with the newlines here. If everything appears good in the browser, ignore this.

**Review by dilirity — DISMISSED** [review]
Looks good to merge. Add the changelog and :shipit:

File: projects/plugins/boost/app/features/image-guide/src/ui/Popup.svelte (hunk 1 above)
Comment (pyronaur) [reply projects/plugins/boost/app/features/image-guide/src/ui/Popup.svelte]: Fixed.

**Review by dilirity — APPROVED** [review]
:shipit: :)

**Pull request merged**